samples/wire/README.md

```diff
@@ -0,0 +1,317 @@
+# Wire Tutorial
+
+Let's learn to use Wire by example. The [Wire README][readme] provides thorough
+documentation of the tool's usage. For readers eager to jump right in, the
+[guestbook sample][guestbook] uses Wire to initialize its components. Here we
+are going to build a small greeter CLI to understand how to use Wire. The
```

Comment:
Consider using the term "program" here instead of "CLI". I know CLI is a fairly common acronym, but I think expanding it out helps inclusivity.

Reply:
The same thought occurred to me, as well. Done.